### DISCUSSION TITLE
Export of articles don´t finnish or verry slow

### DISCUSSION THREAD

**Discussion by exeqtwasnotfree** [atrocore/atrocore#931] [General]
Hello
I have an export job running every morning that exports about 2350 SKUs and with 30 fields/attributes.
Until November 13, this export took about 11-13 minutes.
After this date I have not managed to run a full export, several attempts I have to abort after 8 hours of running.
I manage to run it right now with only 584 SKUs, and it's 48 min.

I think the export failed since the automatic update this weekend.

Are there any known issues with versions core 1.7.6 pim 1.10.1 or export 1.7.35 that can explain this?

**Comment by AnnZast** [comment]
Hello,
You're right, it's related to updates. We are fixing it

**Reply by AnnZast** [reply]
Please update your system and try exporting again. The speed should increase

**Comment by exeqtwasnotfree** [comment]
Thank you AnnZast for info and fast response!
I updated and the job now finished in 3 minutes with 2700 SKU.
